geopandas/geodataframe.py

```diff
@@ -754,7 +763,9 @@ def __geo_interface__(self):
         ``FeatureCollection``.
 
         This differs from `_to_geo()` only in that it is a property with
-        default args instead of a method
+        default args instead of a method.
+
+        CRS of the dataframe is not passed on to the output, unlike `to_json()`.
```

Comment:
```suggestion
        CRS of the dataframe is not passed on to the output, unlike :meth:`~GeoDataFrame.to_json()`.
```
This will add a link to the documentation of `to_json`.

Reply:
You may need to break the line, flake8 will likely complain if you just accent the suggestion.